script/openqa-bootstrap

```diff
@@ -5,10 +5,15 @@ dbname="${dbname:="openqa"}"
 dbuser="${dbuser:="geekotest"}"
 
 # add extra repos for leap
+foundrepo() {
+    zypper lr "$1" && return
+    return 1
+}
+
 source /etc/os-release
-if [[ "$NAME" == "openSUSE Leap" ]] ; then
-	zypper -n addrepo -p 90 obs://devel:openQA devel:openQA
-	zypper -n addrepo -p 91 obs://devel:openQA:Leap:${VERSION} devel:openQA:Leap:${VERSION}
+if [[ "$NAME" == "openSUSE Leap"]] ; then
+        if ! foundrepo devel:openQA; then zypper -n addrepo -p 90 obs://devel:openQA devel:openQA; fi
+        if ! foundrepo devel:openQA:Leap:${VERSION}; then zypper -n addrepo -p 91 obs://devel:openQA:Leap:${VERSION} devel:openQA:Leap:${VERSION}; fi
```

Comment:
While I think this is a nice idea please keep the above change for a separate PR. The challenge is that we want to keep `script/openqa-bootstrap` as easy as possible to read for humans to act as "instructions". Also you are now mixing tabs and spaces. Better to keep the existing style. I have a change prepared to use a consistent spaces-only file as we also have for other files to fix this though.